src/components/pages/courses-page/CoursesPage.module.scss

```diff
@@ -0,0 +1,32 @@
+@import 'src/styles/partials/utils';
+
+.content{
+  @include flex-center;
+  flex-direction: column;
+  gap: 15px;
+  margin-top: 10px;
+}
+
+.header{
+  font-weight: 800;
+  text-align: center;
+}
+
+.description{
+  font-weight: 500;
+  text-align: center;
+  margin-bottom: 20px;
```

Comment:
In order to structure `.scss` file it's better to write all position and boxing attributes at first. In this example: `margin-bottom` should be written first.

Reply:
fixed